spec/support/dataset_helper.rb

```diff
@@ -7,11 +7,12 @@ def click_change(property)
     title: 0,
     summary: 1,
     additional_info: 2,
-    licence: 3,
-    location: 4,
-    frequency: 5,
-    datalinks: 6,
-    documentation: 7
+    topic: 3,
```

Comment:
Could we do a quick change to make this an ordered array and lose the page numbers?

Reply:
Done